CMSgov/easi-app
# [EASI-3633] IT Gov task list closed state

# EASI-3633

[Figma](https://www.figma.com/file/ChzAP34A2DVvQUNQwD7lCt/IT-Governance-Next?type=design&node-id=1256-123186&mode=design&t=Pu2bTGt4tgDnJKNa-0)

## Changes and Description

- Added "Request closed" alert to top of task list
  - Alert only displays if decision is NOT been issued
- Disabled buttons to forms (intake request, draft business case, and final business case)
- Unit tests



## How to test this change

1. Log in as IT Gov admin
2. Complete action to close request without decision (/governance-review-team/:systemId/resolutions/close-request)
3. Go to task list (/governance-task-list/:systemId)
4. Make sure "Request closed" alert is shown and form buttons are disabled
5. Complete action to issue decision on a different request (governance-review-team/:systemId/resolutions/issue-lcid)
6. Go to task list
7. Make sure "Decision issued" alert is displayed in place of "Request closed" alert



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
